gtfu/src/main/java/gtfu/tools/CheckFeedExpiration.java

```diff
@@ -0,0 +1,144 @@
+package gtfu.tools;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import gtfu.ConsoleProgressObserver;
+import gtfu.Debug;
+import gtfu.EmailFailureReporter;
+import gtfu.SlackFailureReporter;
+import gtfu.FailureReporter;
+import gtfu.FeedInfo;
+import gtfu.ProgressObserver;
+import gtfu.Recipients;
+import gtfu.Util;
+import gtfu.Time;
+import java.util.Date;
+
+/**
+ * Daily checks and updates on each active agency's feed.
+ */
+public class CheckFeedExpiration {
+    private static final long MAX_FEED_EXPIRATION_WINDOW_DAYS = 14;
+
+     /**
+     * Runs CheckFeedExpiration for a single agency
+     * @param agencyID The agencyID
+     */
+    public static void CheckFeedExpiration(String agencyID, boolean sendEmailAlert, boolean sendSlackAlert) throws Exception {
```

Comment:
methods should start with a lowercase character (more than one occurrence in this file.

Reply:
done thanks!